examples/docker/Dockerfile.alpine

```diff
@@ -30,7 +30,7 @@ FROM alpine:3.12
 
 COPY . /usr/src/confluent-kafka-python
 
-ENV LIBRDKAFKA_VERSION v1.8.2
+ENV LIBRDKAFKA_VERSION v1.10.0-RC1
```

Comment:
Use v1.9.0-RC10 here to avoid breaking builds. There's hopefully nothing required from a future librdkafka